CDS/cdsresponder/cdsresponder.rb

```diff
@@ -187,8 +188,13 @@ def ThreadFunc
 
           triggerfile=OutputTriggerFile(trigger_content, msg.id)
 
-          @pid = spawn("cds_run", "--route", @routefile, "--#{@cdsarg}", triggerfile)
-          msg=FinishedNotification.new(@routename, $?.exitstatus, GetLogfile(msg.id))
+          cmdline = "cds_run --route \"#{@routefile}\" #{@cdsarg} #{triggerfile}"
+          @logger.debug("Commandline is #{cmdline}")
+          @pid = spawn(cmdline)
+
+          exitstatus = Process.wait @pid
```

Comment:
not sure what the consequences of making this synchronous are... as long as you're aware i'm cool with it ^_^